src/tools/getdns_server_mon.c

```diff
@@ -1587,7 +1587,7 @@ static exit_value test_out_of_order(struct test_info_s *test_info,
         /* A set of asynchronous queries to send. One exists. */
         const char GOOD_NAME[] = "getdnsapi.net";
         struct async_query async_queries[] = {
-                { "1000.delay.getdnsapi.net", 0, false },
+                { "400.delay.getdnsapi.net", 0, false },
```

Comment:
delaydns (the server that answers these queries) only looks at the leftmost label.  I believe detection might become more reliable and less sensitive to caching if we would query for "400.<random label>.delay.getdnsapi.net".  Consider same cache having IPv4 and IPv6 address.  Or serve stale?